config/initializers/cors.rb

```diff
@@ -0,0 +1,20 @@
+# Configure Rack::Cors https://github.com/cyu/rack-cors
+Rails.application.config.middleware.insert_before 0, Rack::Cors, logger: (-> { Rails.logger }) do
+  allow do
+    # Allow all domains access to jobs API
+    origins '*'
+
+    resource '/api/v1/jobs/*',
+      headers: :any,
+      methods: [:get, :post, :delete, :put, :patch, :options, :head]
```

Comment:
Should we allow all methods here? I don't think there are any actions associated with them either way, but just a thought!

Reply:
Yeah it's a good question. I guess leaving it this way is safe even when we introduce actions associated with these, because if we're doing that it means we want external services to be able to use our API to e.g. post. I have just listed all possible methods as this is a public API and we want it to be easy for us to change it.

Reply:
Let's limit it to just the verb(s) we need to serve for now, please.